networkit/cpp/graph/TopologicalSort.cpp

```diff
@@ -4,40 +4,76 @@
  *  Created on: 22.11.2021
  *      Author: Fabian Brandt-Tumescheit
  */
+#include <stdexcept>
+#include "networkit/graph/GraphTools.hpp"
```

Comment:
Sorry for the tedious comment. Please change the include to: `#include <networkit/graph/GraphTools.hpp>`